MergerService/src/run.cs

```diff
@@ -109,6 +109,12 @@ public void Start()
             var pollingTime = this._configurationManager.GetConfiguration<int>("TASK", "pollingTime");
 
             var taskTypes = BuildTypeList();
+            if (taskTypes.Count == 0)
+            {
+                this._logger.LogInformation("No tasks configured, please provide job and task types");
```

Comment:
that's critical error it is not info and it should probably throw and kill the app